### ISSUE TITLE
[ReactUI] Bug in BE with Options POST API

### ISSUE THREAD

**Issue by LaustinSpayce** [Talishar/Talishar#377]
Response from BE for options is something like:
```
{"Settings":[{"name":"HoldPrioritySetting","value":"0"},{"name":"TryReactUI","value":"1"},{"name":"DarkMode","value":"0"},{"name":"ManualMode","value":"0"},{"name":"SkipARWindow","value":"0"},{"name":"SkipDRWindow","value":"0"},{"name":"SkipNextDRWindow","value":"0"},{"name":"AutoTargetOpponent","value":"1"},{"name":"ColorblindMode","value":"0"},{"name":"ShortcutAttackThreshold","value":"0"},{"name":"EnableDynamicScaling","value":"0"},{"name":"MuteSound","value":"0"},{"name":"CardBack","value":"0"},{"name":"IsPatron","value":"0"},{"name":"MuteChat","value":"0"},{"name":"DisableStats","value":"0"},{"name":"IsCasterMode","value":"0"},{"name":"IsStreamerMode","value":"0"}]}
```

lovely. I need to know what the enums are for this (HoldPrioritySetting = 1 is what? for instance)

I need an API to POST / submit updated options to the BE so all the settings can be updated in one request!

**Comment by OotTheMonk** [comment]
Fixed in revision 6f80170d. Was caused by two deprecated settings that aren't used anymore.

**Comment by OotTheMonk** [comment]
No additional FE changes needed; was all on the BE.
